zulip_bots/zulip_bots/bots/virtual_fs/test_virtual_fs.py

```diff
@@ -19,6 +20,20 @@ class TestVirtualFsBot(BotTestCase):
                 '```\n'
                 'Use commands like `@mention-bot help write` for more details on specific\ncommands.\n')
 
+    def test_multiple_recipient_conversation(self) -> None:
+        expected = [
+            ("mkdir home", "foo@example.com:\ndirectory created"),
+        ]
+        message = dict(
+            display_recipient=[{'email': 'foo_stream'}, {'email': 'multi_stream'}],
```

Comment:
Thinking about this, there isn't such a thing as "multi_streams" - if display_recipient is a list of multiple recipients, those recipients are all persons. Can you change the email entries accordingly?